components/DesaDigital/Detail/index.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div class="desa-digital-detail">
+    <div class="desa-digital-detail__action">
+      <BaseButton variant="primary" @click="goBack">
+        <template #icon>
+          <div class="desa-digital-detail__action-button">
+            <div class="desa-digital-detail__action-icon">
+              <IconArrowLeft />
+            </div>
+            Kembali
+          </div>
+        </template>
+      </BaseButton>
+    </div>
+    <div class="desa-digital-detail__content">
+      <h1 class="desa-digital-detail__content-header">
+        Identitas Pemohon
+      </h1>
+      <jds-simple-table class="desa-digital-detail__content-table">
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Nama Lengkap</strong>
+              <span>:</span>
+            </td>
+            <td>{{ item.properties.applicant.name || '-' }}</td>
+          </tr>
+          <tr>
+            <td>
+              <strong>No HP</strong>
+              <span>:</span>
+            </td>
+            <td>{{ item.properties.applicant.phone_number || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Email</strong></td>
+            <td>{{ item.properties.applicant.email || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+    </div>
+    <div class="desa-digital-detail__content">
+      <h1 class="desa-digital-detail__content-header">
+        Detail Desa {{ item.village.name || '' }}
+      </h1>
+      <jds-simple-table class="desa-digital-detail__content-table">
+        <thead>
+          <tr>
+            <th>Tentang Desa</th>
+            <th />
```

Comment:
I wonder why use multiple `th` instead of `colspan`? 🤔

Reply:
Thanks for the quest, I just remember we can expand multiple cells using `colspan` or `rowspan`. The change [here](https://github.com/jabardigitalservice/dedi-cms/pull/92/commits/7b8e47d70a6896540cb6ac6b67eef84f4b155574)